### PR TITLE
[14.0][IMP] sale_order_line_discount_validation: add test cases

### PR DESCRIPTION
**Pull request by csil300**

### CODE REVIEW

**Comment by rafaelbn** [comment]
Dear @csil300 thank you for this great contribution. Please could you check https://github.com/OCA/sale-workflow/pull/1908#issuecomment-1035641291 and sign CLA? Thank you

Dear @rousseldenis this looks like a good improvement adding just test, what do you think? Thank you

File: sale_order_line_discount_validation/tests/test_sale_order_line_discount_validation.py [hunk 1]
```diff
@@ -0,0 +1,86 @@
+from odoo import fields
+from odoo.tests.common import TransactionCase
+
+
+class TestSaleOrderLineDiscountValidation(TransactionCase):
+    def setUp(self):
```
Comment (rousseldenis) [inline sale_order_line_discount_validation/tests/test_sale_order_line_discount_validation.py]: @csil300 Thanks for this. Could you change to SavepointCase as it is faster and change setup to setUpClass ? Thanks

**Review by rousseldenis — CHANGES_REQUESTED** [review]

**Comment by max3903** [comment]
> Hey @csil300, thank you for your Pull Request.
> 
> It looks like some users haven't signed our **C**ontributor **L**icense **A**greement, yet. You can read and sign our full Contributor License Agreement here: http://odoo-community.org/page/cla Here is a list of the users:
> 
> * @csil300 (login unknown in OCA database)
> 
> Appreciation of efforts, OCA CLAbot

Fixed

**Comment by rafaelbn** [comment]
After @csil300 answer to https://github.com/OCA/sale-workflow/pull/1908#discussion_r810849243 we can go ahead!

**Pull request closed without merging**